### ISSUE TITLE
Flask-RESTful is unmaintained

### ISSUE THREAD

**Issue by AndrewEckart** [ssl-hep/ServiceX_App#50]
Per flask-restful/flask-restful#883, the library is no longer being maintained. Upstream issues affecting us such as vimalloc/flask-jwt-extended#86 (errors are not handled correctly when decoding a JWT) may never be fixed. Proposal is to move to the recommended alternative, [Flask-RESTX](https://github.com/python-restx/flask-restx).

**Comment by cnweaver** [comment]
Makes sense to me if it looks like Flask-RESTX will be supported over the next few years.
